### PR TITLE
Removed auto refresh

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Since we want to move to either using sockets or ajax there's no point in keeping these even for now as they cause more harm than good.

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: gui/slick/js/new/comingEpisodes.js [hunk 1]
```diff
@@ -38,8 +38,3 @@ $(document).ready(function(){
         });
     }
 });
-
-setTimeout(function () {
-    "use strict";
-    location.reload(true);
```
Comment (fernandog) [inline gui/slick/js/new/comingEpisodes.js]: @OmgImAlexis can't we keep the comingEpisodes lol ?

Comment (OmgImAlexis) [inline gui/slick/js/new/comingEpisodes.js]: If you need auto refresh install an extension for your browser or hit F5, this causes more problems than it solves and honestly I doubt 99% of users want it. I for one get annoyed as fuck when it randomly refreshes as I'm looking at the page.

**Comment by fernandog** [comment]
https://github.com/SiCKRAGETV/sickrage-issues/issues/3030

File: gui/slick/js/new/errorlogs.js [hunk 2]
```diff
@@ -1,4 +0,0 @@
-setTimeout(function () {
```
Comment (MGaetan89) [inline gui/slick/js/new/errorlogs.js]: Looks like this file can be removed completely, no?

Comment (OmgImAlexis) [inline gui/slick/js/new/errorlogs.js]: @MGaetan89 considering I'm adding ajax for it I decided to leave it for now.

**Pull request merged**